### PR TITLE
support prefixed

### PR DESCRIPTION
**Pull request by evilebottnawi**
Close https://github.com/stylelint/stylelint/issues/1437

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 - Changed: `comment-word-blacklist` no longer ignores words within copyright  comments
 - Changed: `font-family-name-quotes` now checks the `font` property in addition to the `font-family` property
 - Changed: `unit-*` rules now check `@media` values too.
+- Changed: better handling of vendor prefixes in `keyframe-declaration-no-important`, `no-unknown-animations`, `time-no-imperceptible`.
```
Comment (jeddy3) [inline CHANGELOG.md]: Can we break this into three:

- Changed: `keyframe-declaration-no-important` now checks vendor prefixed `@keyframes` at-rules.
- Changed:  `time-no-imperceptible` now checks vendor prefixed properties.
- Fixed: `no-unknown-animations` now classifies vendor prefixed `@keyframes` at-rules are known.

**Comment by jeddy3** [comment]
@evilebottnawi Thanks for tackling these remaining `7.0.0` issues! The `7.0.0` release feels like it's really  close :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,11 +20,14 @@
 - Changed: `comment-word-blacklist` no longer ignores words within copyright  comments
 - Changed: `font-family-name-quotes` now checks the `font` property in addition to the `font-family` property
 - Changed: `unit-*` rules now check `@media` values too.
+- Changed: `keyframe-declaration-no-important` now checks vendor prefixed `@keyframes` at-rules.
+- Changed:  `time-no-imperceptible` now checks vendor prefixed properties.
```
Comment (davidtheclark) [inline CHANGELOG.md]: Extra space.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,11 +20,14 @@
...
 - Added: non-standard syntaxes are automatically inferred from file extensions `.scss`, `.less`, and `.sss`.
 - Added: [processors](/docs/user-guide/configuration#processors.md).
 - Added: `media-feature-parentheses-space-inside` rule.
 - Added: `no-missing-end-of-source-newline` rule.
 - Fixed: The string options of `comment-word-blacklist` will now identify strings *within* comments.
+- Fixed: `no-unknown-animations` now classifies vendor prefixed `@keyframes` at-rules are known.
```
Comment (davidtheclark) [inline CHANGELOG.md]: ... `@keyframes` at-rules as known.

("as" instead of "are")

**Comment by davidtheclark** [comment]
Other than a couple of changelog edits, looks good.

**Comment by davidtheclark** [comment]
LGTM

**Comment by jeddy3** [comment]
Good job.

**Pull request merged**